functions/Import-DbaCsv.ps1

```diff
@@ -439,7 +439,7 @@ function Import-DbaCsv {
                 $elapsed = [System.Diagnostics.Stopwatch]::StartNew()
                 # Open Connection to SQL Server
                 try {
-                    $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential -StatementTimeout 0 -MinimumVersion 9
+                    $server = Connect-DbaInstance -SqlInstance $instance -SqlCredential $SqlCredential -Database $Database -StatementTimeout 0 -MinimumVersion 9
```

Comment:
Please revert to using the `Connect-SqlInstance` command. It has a `-Database` parameter.

I do recall though that we didn't use that in this command for some reason but I've completely forgotten now why.

Reply:
Sure, no problem. I switched it back to Connect-SqlInstance. There is a boundary case that I added a check for in Connect-SqlInstance. Maybe that is the scenario that was observed back when the Import-DbaCsv command was being written. It could explain why the Import-DbaCsv command previously had separate code to check if the database existed (rather than letting Connect-SqlInstance/Connect-DbaInstance handle that).  

On a separate note, is there a long term plan to remove Connect-SqlInstance since it just uses Connect-DbaInstance? It might reduce maintenance effort/cost.

btw, Happy New Year!

Reply:
No. It stays because it is for we control internal use cases with commands that can hit Azure SQL. There is no reason to put that in a public command. It is just a wrapper now for the public function.